docs/plans/2026-02-16-policy-import-export-design.md

```
@@ -0,0 +1,189 @@
+    pub long_break: u32,
+    pub pomodoros_before_long_break: u32,
+    pub parallel_break_policy: String,
+    pub custom_schedule: Option<Schedule>,
+}
+```
+
+## Compatibility Rules
+
+### Version Comparison
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)